dashboard/app/models/section.rb

```diff
@@ -64,6 +64,10 @@ def assign_code
     self.code = unused_random_code
   end
 
+  def teacher_dashboard_url
+    CDO.code_org_url "/teacher-dashboard#/sections/#{id}/manage", 'http:'
```

Comment:
Yes. Note however that the rails URLs generate with `http`, and on all our hosted environments (anything besides development), ssl is enforced so even opening a `http:` will redirect to `https`